account_lock_date_update/wizards/account_update_lock_date.xml

```diff
@@ -9,14 +9,11 @@
         <field name="model">account.update.lock_date</field>
         <field name="arch" type="xml">
             <form>
-                <header/>
-                <sheet>
-                    <group>
-                        <field name="company_id" invisible="1"/>
-                        <field name="period_lock_date"/>
-                        <field name="fiscalyear_lock_date"/>
-                    </group>
-                </sheet>
+                <group>
+                    <field name="company_id" invisible="1"/>
+                    <field name="period_lock_date"/>
+                    <field name="fiscalyear_lock_date"/>
+                </group>
```

Comment:
I didn't realize the ```sheet``` tag made the wizard larger. But about the header, I prefer to declare it even if it's empty in case a submodule wants to add things in it.

Also note that [```account_permanent_lock_move_update```](https://github.com/OCA/account-financial-tools/blob/10.0/account_permanent_lock_move_update/wizards/account_update_lock_date.xml) inherits from this form view.

Reply:
Yes, I started to have a look at account_permanent_lock_move_update. But in term of usability, when you add account_permanent_lock_move_update, it starts to be very strange. It adds a readonly date field (the 2 others are rw) and an additionnal button next to it (the button for the 2 others are at the bottom) and this button starts another wizard. I think that, in terms of usability, it needs a full re-design.